checksum.go

```diff
@@ -197,8 +197,15 @@ func (c *Client) checksumFromFile(checksumFile string, src *url.URL) (*fileCheck
 	if err != nil {
 		return nil, err
 	}
-	relpath, err := filepath.Rel(filepath.Dir(checksumFileURL.Path), absPath)
-	if err != nil {
+	checksumFileDir := filepath.Dir(checksumFileURL.Path)
+	relpath, err := filepath.Rel(checksumFileDir, absPath)
+	switch {
+	case err == nil ||
+		err.Error() == "Rel: can't make "+absPath+" relative to "+checksumFileDir:
```

Comment:
One issue with these comparisons on Windows in particular is that windows error messages are localized. So this will only work for English-language Windows installations.

Reply:
That one is from Rel: https://github.com/golang/go/blob/03a9f5a192149dc6f443eb0e1cee48cb4dd7e26f/src/path/filepath/path.go#L285

 I reaaaly wish this error had a type though ...

Reply:
Do you think there is a better way for this one ?